packages/muelu/src/Rebalancing/MueLu_IsorropiaInterface_def.hpp

```diff
@@ -134,7 +134,7 @@ namespace MueLu {
     
 
     // 3) create graph of amalgamated matrix
-    RCP<CrsGraph> crsGraph = CrsGraphFactory::Build(nodeMap, 10, Xpetra::DynamicProfile);
+    RCP<CrsGraph> crsGraph = CrsGraphFactory::Build(nodeMap, A->getNodeMaxNumRowEntries()*blockdim, Xpetra::StaticProfile);
```

Comment:
See note above.  Can MueLu actually call Isorropia with a Tpetra matrix?

Reply:
No, MueLu will throw.  See 

https://github.com/trilinos/Trilinos/blob/5c9a2ec8019b310b5b74a22e23fcbea11486f781/packages/muelu/src/Rebalancing/MueLu_IsorropiaInterface_def.hpp#L216-L223

Reply:
@jhux2 Thanks for the clarification!  It's probably better for even the Epetra-based code to use StaticProfile, as long as it has good bounds for each row.